app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+require 'page_setup'
+
+class CommentsController < ApplicationController
+  def create
+    @company = Company.find(params[:company_id])
```

Comment:
again, there's a lot going on in this method and a lot of instance variables set. are these all necessary?